### PR TITLE
Update the number of disk reads required to fetch a proof

### PR DESCRIPTION
**Pull request by danieljperry**
This is a minor clarification of the number of disk reads required to fetch a full proof. I'm still not completely sure if I got it right. Could Harold or Mariano review please?

### CODE REVIEW

File: docs/03consensus/proof-of-space.md [hunk 1]
```diff
@@ -35,9 +35,11 @@ See our [plotting FAQ](https://github.com/Chia-Network/chia-blockchain/wiki/FAQ#
...
 
-Finally, the farmer fetches the whole tree of x-values. This requires one disk read for table 7, two for table 6, four for table 5, etc. The whole process thus requires 64 disk reads, which takes approximately 640 ms on a slow HDD with a 10 ms seek time. The amount of data read is small and is independent of plot size.
+The process of inputting a challenge and outputting a proof involves multiple table lookups. First, the farmer reads the f7 value from table 7, the location of which is determined by reading two _C_ tables. After a matching f7 value is found, the farmer reads a single _line point_ in table 7, which contains two _back pointers_ to table 6, four to table 5, eight to table 4, sixteen to table 3, and thirty-two to table 2. This results in a total of 64 disk reads (1 f7 value and 63 line points) being performed to fetch the whole tree of 64 x-values. The 64 disk reads require approximately 640 ms on a slow HDD with a 10 ms seek time. The amount of data read is small and is independent of plot size.
```
Comment (harold-b) [inline docs/03consensus/proof-of-space.md:40]: A few minor corrections here:
After the f7 is found, the value read from T7 is a single index of k+1 bits. These indices are serialized as "park 7" which are just bit-packed entries but not compressed as a line point. 

These P7 index is an index into Table 6, which does contain line points, as do the rest of the tables all the way back as you specified.

**Review by hoffmang9 — DISMISSED** [review]
lgtm

**Review by harold-b — APPROVED** [review]
lgtm

**Review by hoffmang9 — APPROVED** [review]
lgtm

**Comment by hoffmang9** [comment]
Needs conversations resolved before it can be merged

**Pull request merged**